### ISSUE TITLE
Support for RethinkDB stats

### ISSUE THREAD

**Issue by PassTheMayo** [firehol/netdata#3422]
I've seen that netdata allows use of multiple database stats, but RethinkDB is not there. Is there any way that RethinkDB could be added? I use it on a daily basis, and it would be great to see some stats.

**Comment by ktsaou** [comment]
Hi.
You are the first to ask for it. 

Is rethinkdb capable for time-series data? I mean, netdata will flood it with data, especially if multiple netdata servers are pushing metrics to it. From the rethinkdb description, it is not clear if this is its purpose. Do you know?

**Comment by PassTheMayo** [comment]
I'm very sure that RethinkDB allows real time streaming on changes. It's a method in the driver called `.changes()` which streams real-time data changes from databases. You can use this in combination within the internal `rethinkdb` database, and watch for stat changes. RethinkDB records a lot of specific information like table sizes, bytes in, bytes out, other networking info, etc. @ktsaou

**Comment by ktsaou** [comment]
I have probably understood this wrong, so help me out.

I understood that in order for rethinkdb to work, netdata would have to push (`insert`) data to it. This will be huge: 2k metrics per second per server on the average. rethinkdb would need to store them in an efficient way and probably apply some retention policy to them.

Am I wrong?

**Comment by PassTheMayo** [comment]
@ktsaou No, netdata does not need to insert any data into the RethinkDB. It simply just streams data from the internal RethinkDB database into the client, which can be interpreted by netdata.

**Comment by l2isbad** [comment]
@PassTheMayo wants to monitor RethinkDB.
https://rethinkdb.com/docs/system-stats/

**Comment by ktsaou** [comment]
Oh, you mean you want to monitor metrics exposed by rethinkdb using netdata. Sorry, I thought you wanted to use rethinkdb as a backend for archiving netdata metrics.

So, we need @l2isbad for this.

**Comment by ktsaou** [comment]
oops! here he is. yes I got it now.

**Comment by PassTheMayo** [comment]
Oh, yes, I want to monitor RethinkDB's data. Sorry for the confusion.
